src/js/controllers/tx-details.js

```diff
@@ -109,27 +92,21 @@ angular.module('copayApp.controllers').controller('txDetailsController', functio
     }
     if ($scope.btx.note && $scope.btx.note.body) opts.defaultText = $scope.btx.note.body;
 
-    popupService.showPrompt(wallet.name, gettextCatalog.getString('Memo'), opts, function(text) {
+    popupService.showPrompt($scope.wallet.name, gettextCatalog.getString('Memo'), opts, function(text) {
       if (typeof text == "undefined") return;
 
+      $scope.btx.note.body = text;
```

Comment:
Transactions without notes will crash in this line, should be defined as:

>var note = {
        body: text
      };
      $scope.btx.note = note;